Simplenote/SimplenoteAppDelegate+Swift.swift

```diff
@@ -237,6 +238,24 @@ extension SimplenoteAppDelegate {
 
         SPTracker.trackSettingsStatusBarDisplayMode(hidden: Options.shared.statusBarHidden)
     }
+
+    @objc
+    func handleUserActivity(_ userActivity: NSUserActivity) -> Bool {
+        if userActivity.activityType == "com.apple.corespotlightitem" {
```

Comment:
This is a very simple implementation of handle User activity for now, but one of the other things I would like to do in the near future is bring SNMac into parity with SNiOS as far as shortcuts are concerned.  When I do that this will need to be expanded, but for now we aren't handling any other user activity types

Reply:
I think you can use the `CSSearchableItemActionType` constant instead of `com.apple.corespotlightitem`:

```
(lldb) po CSSearchableItemActionType
"com.apple.corespotlightitem"
```